docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,80 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Improve performance of queries with grouping on joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the spooling protocol.
```

Comment:
There's no user visible change other than renaming some properties. The new protocol has been in Trino for a few releases now.

Reply:
@martint yes, but wasn't called out and documented

Reply:
Yeah .. and we never announced it or added anything in any release notes.. we were waiting until docs are in and it is stable.. that is now.. cc @wendigo